### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by Arthusppp**
Some PL translations.

Can someone take a look and say how to use the plural form? I am unable to fix the translation for the Scouting and Mysticism secondary skill. (I mean, the translation in the file is correct; however, I cannot see the translated text on my local env).

### CODE REVIEW

**Comment by ihhub** [comment]
Hi @Arthusppp , for plural forms issue we have #3919 ticket. It's still unsolved.

File: files/lang/pl.po [hunk 1]
```diff
@@ -4997,142 +4989,121 @@ msgstr "Zaawansowany Majątek"
...
-#, fuzzy
 msgid "%{skill} increases your hero's viewable area by one square."
 msgid_plural ""
 "%{skill} increases your hero's viewable area by %{count} squares."
-msgstr[0] "Zwiększa pole widzenia bohatera o %{count} kwadrat."
-msgstr[1] "Zwiększa pole widzenia bohatera o %{count} kwadraty."
-msgstr[2] "Zwiększa pole widzenia bohatera o %{count} kwadratów."
+msgstr[0] "Zwiększa pole widzenia bohatera o %{count}."
+msgstr[1] "Zwiększa pole widzenia bohatera o %{count}."
+msgstr[2] "Zwiększa pole widzenia bohatera o %{count}."
```
Comment (ihhub) [inline files/lang/pl.po:5014]: Why did we remove squares here?

File: files/lang/pl.po [hunk 2]
```diff
@@ -4997,142 +4989,121 @@ msgstr "Zaawansowany Majątek"
...
 
-#, fuzzy
 msgid ""
 "%{skill} gives your hero a %{count} percent chance to learn any given 4th "
 "level spell (or below) used against him in combat."
 msgstr ""
 "Daje Twojemu bohaterowi %{count} procent szans na nauczenie się dowolnego "
-"czaru 4-go poziomu (lub poniżej) użytego przez wroga w kierunku Twojego "
-"bohatera."
+"czaru 4-go poziomu (lub poniżej)  rzucanego przez przeciwnika. "
```
Comment (ihhub) [inline files/lang/pl.po:5095]: Please remove 1 space before `rzucanego` word.

**Review by ihhub — COMMENTED** [review]
Hi @Arthusppp , I left few comments here. Would you mind to take a look?

File: files/lang/pl.po (hunk 1 above)
Comment (Arthusppp) [reply files/lang/pl.po:5014]: In Polish, both mean exactly the same. However, the OG uses the new one.

File: files/lang/pl.po (hunk 2 above)
Comment (Arthusppp) [reply files/lang/pl.po]: Done.

**Comment by ihhub** [comment]
@Arthusppp , thank you very much for this change!

**Pull request merged**